backend/src/data/repositories/user-details/user-details.repository.ts

```diff
@@ -29,7 +29,9 @@ class UserDetails {
         userId,
       })
       .withGraphFetched('avatar')
-      .returning('*')
+      .returning(
+        'id, gender, createdAt, updatedAt, dateOfBirth, userId, fullName, avatarFileId, telegramUsername',
```

Comment:
If all this cols are being defined in different places for a few times (here and also in model's modifier) won't it be better to move them inside one const array and use from it?